config/locales/es.yml

```diff
@@ -48,8 +114,42 @@ es:
   pagination:
     next: Próximo
     prev: Anterior
+  remote_follow:
+      acct: Entra tu usuario@dominio que tu quieres seguir desde
+      missing_resource: No se ha encontrado la URL requerida para tu cuenta
+      proceed: Proceder a seguir
+      prompt: 'Tu estas apunto de seguir a:'
   settings:
+    authorized_apps: Aplicaciones Autorizadas
+    back: Volver a Mastodon
     edit_profile: Editar perfil
+    export: Exportar información
     preferences: Preferencias
+    settings: Ajustes
+    two_factor_auth: Two-factor Authentication
+  statuses:
+    open_in_web: Abrir en la web
+    over_character_limit: limite de carácteres, %{max} excedidos
+    visibilities:
+      private: Solo mostrar a tus seguidores
+      public: Público
+      unlisted: Público pero no mostrar en la timeline pública
+  stream_entries:
+    click_to_show: Mostrar
+    reblogged: boosted
+    sensitive_content: Contenido sensible
+  time:
+    formats:
+      default: "%b %d, %Y, %H:%M"
+  two_factor_auth:
+    description_html: Si tu activas el <strong>two-factor authentication</strong>, te enviaremos un token al móvil por cada inicio de sesión.
+    disable: Desactivar
+    enable: Activar
+    instructions_html: "<strong>Escanea este código QR en una aplicación como Google Authenticator o alguna similar</strong>. Esta aplicación te generara los tokens."
+    plaintext_secret_html: 'Plain-text secret: <samp>%{secret}</samp>'
```

Comment:
See https://es.wikipedia.org/wiki/Archivo_de_texto for a list of way to translate plain-text